src/effects/VST/VSTEffect.h

```diff
@@ -101,6 +105,33 @@ struct VSTEffectWrapper : public VSTEffectLink
    using ParameterVisitor = std::function< bool(const ParameterInfo& pi) >;
 
    void ForEachParameter(ParameterVisitor visitor) const;
+
+   // These are here because they are used by the import/export methods
+   int mVstVersion;
+   wxString mName;
+
+   // XML load/save
+   bool mInSet;
+   bool mInChunk;
+   wxString mChunk;
+   long mXMLVersion;
+   VstPatchChunkInfo mXMLInfo;
+
+   bool LoadXML(const wxFileName& fn);
+   bool HandleXMLTag(const std::string_view& tag, const AttributesList& attrs) override;
+   void HandleXMLEndTag(const std::string_view& tag) override;
+   void HandleXMLContent(const std::string_view& content) override;
+   XMLTagHandler* HandleXMLChild(const std::string_view& tag) override;
+
+   void SetString(int opcode, const wxString& str, int index = 0);
+
+   ComponentInterfaceSymbol GetSymbolB() const
+   {
```

Comment:
Now that mName is in the wrapper, I suggest:
- make this function non-inline
- simply call it `GetSymbol`
- Redefine `VSTEffect::GetSymbol` (a virtual function override) as the call-through to `VSTEffectWrapper::GetSymbol`

Reply:
ok, done